mpmath/tests/test_gammazeta.py

```diff
@@ -508,20 +508,6 @@ def test_gamma_huge_5():
         "7571892628368354580620654233316841")
     mp.dps = 15
 
-def test_gamma_huge_6():
```

Comment:
Why was this removed?

Reply:
The test doesn't look right.  I'm not sure about numerical values used, do you?  And it's disabled, anyway (early "return" statement).

An option may be make this xfailed test.  But only after we make sure that tested values are correct.

Reply:
You're right, the real part is not right. May as well remove it.